plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -142,6 +154,11 @@ SecRule REQUEST_FILENAME "@contains /remote.php/dav/files/" \
     ctl:ruleRemoveById=953100-953130,\
     ctl:ruleRemoveById=920440,\
     ctl:ruleRemoveTargetById=933210;REQUEST_FILENAME,\
+    ctl:ruleRemoveTargetById=920220;REQUEST_URI,\
```

Comment:
Please explain these exclusions in the rule comment as was done for the others. Please order by ID.

Reply:
I think the comments here explain it well enough, should I add "Uploading files"?

Reply:
also, you want me to order the rule IDs highest to lowest or lowest to highest?

Reply:
I went to look at the rule numbers (e.g., 920220) and it wasn't immediately clear why the rules would need to be excluded. I suppose in this case it would be because Nextcloud performs invalid URL encoding?
The reason I'd like the comments is so when we revisit this rule at some point in the future we can decide whether it still makes sense for a given ID to be excluded. If we don't know why it's there we end up guessing. This sort of thing has lead to many problems in the past for this project.

Sort IDs from lowest to highest.

Reply:
I get what you mean now, I'll try and improve the comments for that rule

Reply:
I can't reproduce and find the cause of 920220 FP, maybe the FP doesn't exist in newer Nextcloud version, should I remove it and re-add in the future when I can find the cause? it's only a warning so it's not a big deal.

I can reproduce the request body FPs like 944120 but I'm not sure what comment to put for it, 944120 is triggered by ``https://gamedownloads.rockstargames.com/mtl/js/20220405bcbd/build/launcher.fc6801e3b29d7eae931e.js/*! for license information please see launcher.fc6801e3b29d7eae``, should I put something along the lines of "allow upload of files with funny content"?

Reply:
Yes, remove 920220.

How about "Allow uploads of files containing code (Java, JavaScript, etc.)"?

Reply:
yeah that sounds good to me